lib/widgets/mentionned_user.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:fluffychat/pages/chat/chat.dart';
+import 'package:fluffychat/presentation/extensions/send_file_extension.dart';
+import 'package:fluffychat/utils/string_extension.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_matrix_html/text_parser.dart';
+
+class MentionnedUser extends StatelessWidget {
+  final String identifier;
+  final String url;
+  final OnPillTap? onTap;
+  final TextStyle? textStyle;
+  final ChatController chatController;
```

Comment:
remove `ChatController`, i think `User user` is enough

Reply:
done, I simplified the widget